languages/PL.json

```diff
@@ -0,0 +1,28 @@
+{
+  "outageStrings": {
+    "upString1": "jest z powrotem online!",
+    "upString2": "ma poprawioną wydajność.",
+    "degradedString1": "doświadcza problemy.",
+    "degradedString2": "ma obniżoną wydajność.",
+    "downString1": "jest offline.",
+    "downString2": "ma awarię.",
+    "responseTime": "Czas Odpowiedzi",
+    "httpCode": "Kod Statusu",
+    "incidentId": "Identyfikator incydentu",
+    "machineId": "Identyfikator maszyny Roblox",
+    "responseMessage": "Wiadomość z Odpowiedzią",
+    "errorMessage": "Wiadomość z Błędem"
```

Comment:
This could be changed to "Odpowiedź Serwera" and "Błąd Serwera" so that it means "Server Response/Error" rather than "Message with Response/Error"

Reply:
I based my translation on translations already implemented. This is the way it was written in others.

Reply:
In what others?

Reply:
In ES.json, FR.json, ID.json and PT.json.

Reply:
You based your translation on translations for 4 other languages?
I hope you do realize every language is unique and you can't base one on the others...

Reply:
I did not translate from those languages to Polish, however, due to the fact that there is no way of saying "Error Message" in Polish means you have to use workarounds, such as "Wiadomość z Błędem". You said to use "Błąd Serwera" which means Server Error, not errorMessage or Message Error.
Other languages which also have issues with translating this specific phrase have used a similar workaround to the one I did. They have been approved by Status+ meaning that they aren't bad translations, they're actually the accurate translation.

Reply:
What you say is true, however your workaround of "Message with Response/Error" is worse than something like I suggested.
The word "wiadomość" is less universal in polish and it implies an independent act of someone/something sending it to another someone/something, ex. that the bot suddenly received it, like if it was subscribed to a tracker and just received it, which is not the case. And having it say "z błędem" (with error) emphasizes the message object itself as the subject, saying that it was carrying an error as an extra information.
On the other hand "Błąd Serwera" emphasizes the error as the subject and includes the context that it was encountered when communicating with the server, as in the server threw it and it was not just received randomly.